lib/commons/builder/queries/legislative_index_terms.rq.liquid

```diff
@@ -26,6 +27,31 @@ WHERE {
                                    pq:P2937 ?term ] .
   }
 
+  OPTIONAL {
+    ?house p:P1342 ?numberOfSeatsHouseStatement .
+    ?numberOfSeatsHouseStatement ps:P1342 ?numberOfSeatsHouse .
+    OPTIONAL { ?term (wdt:P580|wdt:P571) ?termStart }
+    OPTIONAL { ?term (wdt:P582|wdt:P576) ?termEnd }
+    OPTIONAL {
+      ?numberOfSeatsHouseStatement pq:P580 ?numberOfSeatsHouseStatementStart .
+    }
+    OPTIONAL {
+      ?numberOfSeatsHouseStatement pq:P582 ?numberOfSeatsHouseStatementEnd .
+    }
+    FILTER (!(BOUND(?numberOfSeatsHouseStatementStart) && BOUND(?termStart) && (?numberOfSeatsHouseStatementStart > ?termStart)))
+    FILTER (!(BOUND(?numberOfSeatsHouseStatementEnd) && BOUND(?termEnd) && (?numberOfSeatsHouseStatementEnd < ?termEnd)))
+    FILTER (!(BOUND(?numberOfSeatsHouseStatementEnd) && BOUND(?termStart) && (?numberOfSeatsHouseStatementEnd < ?termStart)))
+  }
+  OPTIONAL {
+    ?term p:P1342 ?numberOfSeatsTermStatement .
```

Comment:
As I understand it, this query will only use the term-attached seat count if there's also a house-attached seat count. That might be fine, if there's always a house-attached seat count, but I wanted to double-check since the way I read the commit mesage made me think it would use the term-attached seat count even if there's no house-attached seat count.

Reply:
If there's no house-attached seat count, it should still use the term-attached one, which it the behaviour I saw when testing (though I could have overlooked something). Do you think it does different?

http://tinyurl.com/ydxf2cct is the proto-commons-united-kingdom terms query with this branch, stripped down to just the House of Commons, and showing historical terms. The first couple have term-based counts but no applicable house-based statements.

Reply:
Never mind: I was totally misreading the query - I thought one OPTIONAL block was nested in the other, when it isn't at al...